index.rst

```diff
@@ -42,10 +42,12 @@ learned directly - and of course to rewind as often as you like and need.
    message_ix_workshop
    gains_workshops
    pyam
+   message_ix_community_meeting
```

Comment:
I suggest to move the community-meeting to the top of the list, and rename the file `message_ix_workshop` to `message_ix_workshops` (for consistency with the gains page).